waku-bindings/src/node/relay.rs

```diff
@@ -58,43 +55,15 @@ pub fn waku_create_content_topic(
         .expect("&str from result should always be extracted")
 }
 
-/// Default pubsub topic used for exchanging waku messages defined in [RFC 10](https://rfc.vac.dev/spec/10/)
-pub fn waku_default_pubsub_topic() -> WakuPubSubTopic {
-    let mut result: String = Default::default();
-    let result_cb = |v: &str| result = v.to_string();
-    let code = unsafe {
-        let mut closure = result_cb;
-        let cb = get_trampoline(&closure);
-        waku_sys::waku_default_pubsub_topic(cb, &mut closure as *mut _ as *mut c_void)
-    };
-
-    handle_response(code, &result).expect("&str from result should always be extracted")
-}
-
-/// Get the list of subscribed pubsub topics in Waku Relay.
-/// As per the [specification](https://rfc.vac.dev/spec/36/#extern-char-waku_relay_topics)
-pub fn waku_relay_topics() -> Result<Vec<String>> {
-    let mut result: String = Default::default();
-    let result_cb = |v: &str| result = v.to_string();
-    let code = unsafe {
-        let mut closure = result_cb;
-        let cb = get_trampoline(&closure);
-        waku_sys::waku_relay_topics(cb, &mut closure as *mut _ as *mut c_void)
-    };
-
-    handle_json_response(code, &result)
-}
-
 /// Publish a message using Waku Relay
 /// As per the [specification](https://rfc.vac.dev/spec/36/#extern-char-waku_relay_publishchar-messagejson-char-pubsubtopic-int-timeoutms)
 pub fn waku_relay_publish_message(
+    ctx: *mut c_void,
     message: &WakuMessage,
-    pubsub_topic: Option<WakuPubSubTopic>,
+    pubsub_topic: &String,
```

Comment:
In the future we may want to do something like `pubsub_topic: impl Into<WakuPubsubTopic>` and impl `From<String>` for `WakuPubsubTopic` Or even better remove pubsub topic and use shards with the appropriate type constraints.